tests/functional/tests/bundles/base-yoga-ovn.yaml

```diff
@@ -134,10 +134,18 @@ applications:
     charm: ch:nova-compute
     channel: yoga/stable
     num_units: 1
-    storage:
-      ephemeral-device: '40G'
     options:
+      enable-live-migration: true
+      enable-resize: true
+      encrypt: true
+      force-raw-images: true
+      libvirt-image-backend: rbd
+      migration-auth-type: ssh
+      virt-type: lxc
```

Comment:
I'm not sure we need to add all these options. If we can stay as default as possible, that would be simpler.